demos/tests/run_tests.py

```diff
@@ -50,8 +53,26 @@ def parse_args():
         help='list of demos to run tests for (by default, every demo is tested)')
     parser.add_argument('--mo', type=Path, metavar='MO.PY',
         help='Model Optimizer entry point script')
+    parser.add_argument('--device_list', required=False, default="CPU, GPU",
```

Comment:
This should be called `--devices` for consistency with `--demos`. It should also use the same syntax.

Also, `required=False` is redundant.